code/datums/callback.dm

```diff
@@ -68,6 +68,21 @@
 	if(usr)
 		user = WEAKREF(usr)
 
+/**
+ * Qdel a callback datum
+ * This is not allowed and will stack trace. callback datums are structs, if they are referenced they exist
+ *
+ * Arguments
+ * * force set to true to force the deletion to be allowed.
+ * * ... an optional list of extra arguments to pass to the proc
+ */
+/datum/callback/Destroy(force=FALSE, ...)
+	SHOULD_CALL_PARENT(FALSE)
+	if (force)
+		return ..()
+	stack_trace("callbacks can not be qdeleted. if they are referenced they must exist. ([object == GLOBAL_PROC ? GLOBAL_PROC : object.type] [delegate])")
```

Comment:
Nit: We tend to write our stack traces with proper grammar

```suggestion
	stack_trace("Callbacks can not be qdeleted. If they are referenced, they must exist. ([object == GLOBAL_PROC ? GLOBAL_PROC : object.type] [delegate])")
```

Reply:
spelling is a spook